SC_redesign/Crop_and_Soil/field/manure_application.py

```diff
@@ -151,6 +159,10 @@ def _apply_solid_machine_manure(self, dry_matter_mass: float, dry_matter_fractio
             Total mass of phosphorus in this application of manure (kg)
```

Comment:
Missing periods on a handful of these param definitions as well.
```suggestion
            Total mass of phosphorus in this application of manure (kg).
```